x/group/migrations/v2/migrate.go

```diff
@@ -0,0 +1,72 @@
+package v2
+
+import (
+	"encoding/binary"
+	"fmt"
+
+	storetypes "github.com/cosmos/cosmos-sdk/store/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/address"
+	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+	"github.com/cosmos/cosmos-sdk/x/group"
+	"github.com/cosmos/cosmos-sdk/x/group/internal/orm"
+)
+
+const (
+	ModuleName = "group"
+
+	// Group Policy Table
+	GroupPolicyTablePrefix    byte = 0x20
+	GroupPolicyTableSeqPrefix byte = 0x21
+)
+
+// Migrate migrates the x/group module state from the consensus version 1 to version 2.
+// Specifically, it changes the group policy account to from module account to base account.
+func Migrate(
```

Comment:
This PR also needs a genesis JSON migration, but let's do it in a follow-up (and it's not blocking beta & backportable)

Reply:
Sure!